.github/workflows/create-alpha-release-branch.yaml

```diff
@@ -0,0 +1,80 @@
+name: Create Alpha Release Branch
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Alpha Release Branch Version'
+        required: false
+
+env:
+  artifactId: gaffer2
```

Comment:
As it is an environment variable should it be `ARTIFACT_ID`

Reply:
`artifactId` is used because it is used in maven release process for naming the artifact: http://maven.apache.org/guides/mini/guide-naming-conventions.html

Reply:
So is it used externally in a maven process somewhere? If not and it is just used by actions then shouldn't you use the actions env precedence which is capitals with underscores?

Reply:
When there is a maven release (in the release workflow) it is picked up by maven to give the artifact name. Therefore it just seems right to use that across all workflows rather than change it from `ARTIFACT_ID` in create-release-branch to `artifactId` in release

Reply:
The name is reserved in maven as it actually is used to indicate the artifact in the pom. This is an totally different language with different rules

Reply:
When `mvn deploy` is run in the release it grabs `artifactId` from an environment variable and uses it to set the name of the artifact in the pom. https://github.com/gchq/Gaffer/blob/0117e3faf758d87e339bcb2513028b485f03fe1a/.github/workflows/release.yaml#L227

The line in question sets that variable as an environment variable. The reason it is used in all 4 workflows is consistency.

Reply:
From what I can tell `artifactId` is an actions env variable that is injected:
```git commit -a -m "prepare release $artifactId-$RELEASE_VERSION"```
If it already existed before then it should have been flagged up at a previous code review.

Reply:
It did already exist, GitHub actions adds it as an environment variable to the machine that is running the job, so that git commit line correctly uses it, but `mvn deploy` also uses it as it looks for the `artifactId` environment variable.

Reply:
Sorry, still not seeing where `artifactId` is actually being used other than with a `$` which substitutes the env variable...

Reply:
Perhaps you are right, I presumed because it was like this before, `artifactId` was used by maven in the deploy stage but I can't find any evidence that it is. I can change this to `ARTIFACT_ID` and if something goes weird and breaks on the first release run then change them all back I suppose.

Reply:
Even though it is correct for environment variables to be all caps, `artifactId` still looks better to me as it mirrors the pom.